src/main/java/org/apache/maven/plugins/javadoc/resolver/ResourceResolver.java

```diff
@@ -19,26 +19,29 @@
  * under the License.
  */
 
+

```

Comment:
Seems like one empty line too much...

Reply:
Fixed in https://github.com/apache/maven-javadoc-plugin/pull/164/commits/b88e2980334497fbafa0d26dd48fcf136f819dd0.